### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by ghost** [arkypita/LaserGRBL#1024] [Q&A]
I don't know a thing about lasers. I had ordered a cheep Super Carver off the internet. It came with software that worked for 1 day then vanished from my computer. Now I have the Super Carver but no software. I have no idea what the machine is programmed to run but hoping it may be grbl? Can anyone help me with this issue?

**Comment by thecuress** [comment]
TRY to install lasergrbl

**Comment by Schwarzi07** [comment]
Could you please provide more Information about the engraver.

**Comment by thecuress** [comment]
i download the softaware....i try and I tell you if that's right

**Comment by thecuress** [comment]
try this link   https://smartcarve.software.informer.com/download/

**Comment by ghost** [comment]
Schwarzi07... It's a SuperCarver 100W laser. I  did install lasergrbl and on com 3 baud 9600 it homes but wont connect. I tried benbox but none of the .hex files work. Totally new to this and totally confused?

**Comment by ghost** [comment]
thecuress... I tried to download smartcarv but it wants do download an .rar

**Comment by thecuress** [comment]
it's right.....use  winrar or something else to unzip

**Comment by StuartB4** [comment]
Change the Baudrate to 57600.
